src/python/pants/testutil/test_base.py

```diff
@@ -92,6 +93,8 @@ class TestBase(unittest.TestCase, metaclass=ABCMeta):
     :API: public
     """
 
+    additional_options: List[str] = []
```

Comment:
Rather than `options`, this should probably be `args`. It might also be worth calling them `additional_bootstrap_args`, since in cases where tests can inject options as a `Param` (as @Eric-Arellano showed) they should do that instead. Adding a comment explaining that would be good too.